jpa/src/test/java/org/example/app/it/DatabaseIT.java

```diff
@@ -21,46 +21,136 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import java.io.InputStream;
+import java.util.ArrayList;
 import java.util.Collection;
 
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+
+import org.example.app.Person;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.microshed.testing.SharedContainerConfig;
-import org.microshed.testing.jaxrs.RESTClient;
-import org.microshed.testing.jupiter.MicroShedTest;
-import org.example.app.*;
 
-@MicroShedTest
 public class DatabaseIT {
 
-	@RESTClient
-	public static PersonResource personSvc;
+	private static Jsonb jsonb;
+	private static Client client;
+
+	@BeforeAll
+	public static void init() {
+		client = ClientBuilder.newClient();
+		jsonb = JsonbBuilder.newBuilder().build();
+	}
+
+	@AfterAll
+	public static void destroy() throws Exception {
+		jsonb.close();
+		client.close();
+	}
+
+	public WebTarget getTarget(String path) {
+		String port = System.getProperty("http.port");
+		String context = System.getProperty("context.root");
+		String url = "http://localhost:" + port + "/" + context + "/";
```

Comment:
Can we log this `url`.. I just want to show an IT logging sample.. we were using log4j before I assume since MST libs did.   I think using slf4j-log4j12   (so SLF4J with log4j config) would be a fine choice.

Reply:
I can add  the log4j dependency and print the message.

Reply:
Thanks